libvirt/map.jinja

```diff
@@ -144,10 +144,7 @@
 {%-     endif %}
 
 {%-     for map_value in map_values %}
-{%-       set yamlfile = map_sources_dir | path_join(
-            map_source,
-            map_value ~ ".yaml",
-          ) %}
+{%-       set yamlfile = map_sources_dir ~ "/" ~ map_source ~ "/" ~ map_value ~ ".yaml" %}
```

Comment:
If concatenation is the final method selected, then can we keep this in "Black" format by splitting this over multiple lines.  An alternative structure which could be considered is using `.format()` like the following:

```suggestion
{%-       set yamlfile = "{}/{}/{}.yaml".format(
            map_sources_dir,
            map_source,
            map_value,
          ) %}
```

Would be worth testing this on Windows, though!

Reply:
Great, I like it better, it's easier to mangle parameters ;-)